integration_tests/pages/authManageDetails.ts

```diff
@@ -2,6 +2,6 @@ import Page from './page'
 
 export default class AuthManageDetailsPage extends Page {
   constructor() {
-    super('Your account details')
+    super('Your account details', false)
```

Comment:
think this is fine but just to be aware having [boolean params](https://martinfowler.com/bliki/FlagArgument.html) is often seen as a code smell for a few reasons, one being that it's not obvious when reading the code what they mean.  

A way to help in this situation might be passing an object as the boolean then has context of having a  name
e.g:
```
 super('Your account details', { hasBackLink: false })
```